django_ibmi/introspection.py

```diff
@@ -32,7 +33,25 @@ class DatabaseIntrospection(BaseDatabaseIntrospection):
     """
 
     data_types_reverse = {
-        # TODO define reverse data types
+        pyodbc.SQL_CHAR:"CharField",
+        pyodbc.SQL_VARCHAR: "TextField",
```

Comment:
From what I can gather, CharField requires a max_length and is usually a smaller length. Since CHAR, VARCHAR, GRAPHIC, an VARGRAPHIC all require a length field to be given (unlike CLOB and DBCLOB, which have a default size and are can be larger than CHAR/VARCHAR/etc) I think all these types should map to CharField.

This means SQL_CHAR, SQL_VARCHAR, SQL_WCHAR, and SQL_WVARCHAR should map to CharField.

FYI, the driver returns the "long" types for CLOB, DBCLOB, and BLOB.